pages/article.js

```diff
@@ -227,14 +228,14 @@ class ArticlePage extends React.Component {
 
         <section className="section">
           <header className="header">
-            <h2>訊息原文</h2>
+            <h2>訊息原文 - {slicedArticleTitle}...</h2>
```

Comment:
I think h2 is more like article title, but since the sliced title actually not the proper title, so maybe it does not appropriate. I will recover it.
(especially SEO for article page is enough at the beginning.)